config/_default/config.toml

```diff
@@ -105,33 +105,39 @@ hasChildren = true
 
     [[languages.en.menu.main]]
     parent = "Documentation"
-    name = "v3.1.x <img src='/images/header/star.svg' alt='star'>"
+    name = "v3.2.x <img src='/images/header/star.svg' alt='star'>"
     URL = "docs/"
     weight = 1
 
+    [[languages.en.menu.main]]
+    parent = "Documentation"
+    name = "v3.1.x"
```

Comment:
I think we need to keep the `<img>` here. After 3.2 GA, we can move it.
Same for the ZH website.

Reply:
done.